Makefile

```diff
@@ -0,0 +1,13 @@
+OUT := build
+MPYCROSS := /usr/bin/mpy-cross
+MPYC_FLAGS := -v -v -mcache-lookup-bc
+PYS := $(shell find ./ev3dev2 -type f \( -iname "*.py" ! -iname "setup.py" \))
+MPYS := $(PYS:./%.py=${OUT}/%.mpy)
+vpath %.py .
+
+${OUT}/%.mpy: %.py
+	@mkdir -p $(dir $@)
+	${MPYCROSS} ${MPYC_FLAGS} -o $@ $<
+
+install: ${MPYS}
+	cp -R $(OUT)/* /usr/lib/micropython/

```

Comment:
Will ignore the sections of this PR that overlap with #648

Reply:
Yipes. I acknowledge the confusion. This PR and #648 are mutually exclusive... this one creates a separate Console() class that implements the same functionality found in #648 where I integrated it within the Display() class for micropython only. My request of the repo owners: choose one and close the other :-)